Makefile.inc

```diff
@@ -1,4 +1,6 @@
 k8s_version ?= 1.14.6
+olm_version ?= 0.14.1
+olm ?= false
```

Comment:
I think the idea now is to rely only on `CLUSTERS_ARGS` instead of adding Make variables. @mkolesnik what do you think?

Reply:
Done, thanks. Should `k8s_version` be in `CLUSTER_ARGS`? That's what confused me.

Reply:
@mkolesnik sent a patch to fix this here https://github.com/submariner-io/shipyard/pull/177. Thanks for clarifying!